CONTRIBUTING.md

```diff
@@ -201,13 +201,21 @@ Here are some general rules for adding tests:
 * Sometimes flake8 complains about lines that are actually fine, in which case you can put a comment on the line such as: # noqa or # noqa: ANNN, where ANNN is the error code that flake8 issues.
 * To collect output for `CProfile`, run dbt with the `-r` option and the name of an output file, i.e. `dbt -r dbt.cprof run`. If you just want to profile parsing, you can do: `dbt -r dbt.cprof parse`. `pip` install `snakeviz` to view the output. Run `snakeviz dbt.cprof` and output will be rendered in a browser window.
 
-## Adding a CHANGELOG Entry
+## Adding or modifying a CHANGELOG Entry
 
 We use [changie](https://changie.dev) to generate `CHANGELOG` entries. **Note:** Do not edit the `CHANGELOG.md` directly. Your modifications will be lost.
 
 Follow the steps to [install `changie`](https://changie.dev/guide/installation/) for your system.
 
-Once changie is installed and your PR is created, simply run `changie new` and changie will walk you through the process of creating a changelog entry.  Commit the file that's created and your changelog entry is complete!
+Once changie is installed and your PR is created for a new feature, simply run the following command and changie will walk you through the process of creating a changelog entry:
+
+```bash
```

Comment:
I've been using `shell` instead of `sh` or `bash` in order to be both verbose and generic about the specific `<language>` covered by the code being fenced. I've read that `console` also works with some (most?) Markdown parsers, but haven't really used that variant personally.

I don't know if their are best practices for the language specifier 🤷 , but offering my personal default as a suggestion anyways:
```suggestion
```shell
```

Reply:
Here's a cool Markdown style guide we could choose to adopt or modify for our own purposes:
https://arcticicestudio.github.io/styleguide-markdown/